platformio.ini

```diff
@@ -589,7 +589,7 @@ lib_ignore  = LiquidCrystal, LiquidTWI2, Adafruit NeoPixel, TMCStepper, U8glib-H
 platform      = ${common_stm32f1.platform}
 extends       = common_stm32f1
 board         = genericSTM32F103ZE
-extra_scripts = buildroot/share/PlatformIO/scripts/chitu_crypt.py
+#extra_scripts = buildroot/share/PlatformIO/scripts/chitu_crypt.py
```

Comment:
If the script obsolete, or could it be needed in future?

Reply:
To explain:
This script will allow users flash marlin directly from SD. But it's broken now. So we commented out until @xC0000005 fixed that.

He call me a few minutes ago and now he has a working version! We will update the script soon, after a few tests.

Reply:
Excellent…

Reply:
Done!! It's working directly from SD! No need to open the board case and flash it manually!!